### ISSUE TITLE
No Testcase capitalization after a '(' [RenameTestCases]

### ISSUE THREAD

**Issue by ButterBetzi** [MarketSquare/robotframework-tidy#485]
With the following setting: `"RenameTestCases: enabled=True: capitalize_each_word=True"` testcases that have a word preceeded by a '(' do not get capitalized.
Example:
```
Testcasename that isnt Capitalized (but will Be)
``` 
results in:
```
Testcasename That Isnt Capitalized (but Will Be)
```

**Comment by bhirsz** [comment]
It's probably because of the whole nightmare with detecting if given array of characters is a word that we want to capitalize or not.  Current implementation is quite naive:

```
token_value = " ".join(f"{word[0].upper()}{word[1:]}" for word in token.value.split(" "))
``` 
(split name on spaces, every first character in every element should be uppercase). I will think if we want to complicate it and uppercase first letter-like character in a word rather than just first. For now I don't see reason why not, it's not keyword name so there is no risk of capitalizing variable name by mistake.

**Comment by bhirsz** [comment]
Ok, I see that RenameKeywords have different logic:
```
return "".join([a if a.isupper() else b for a, b in zip(value, string.capwords(value))])
```
I will reuse the same logic for consistency (and see if this logic works for your case).

**Comment by bhirsz** [comment]
This change was released together with Robotidy 4.0. However I recommend using Robotidy 4.0.1 when it is (soon) released since I also introduced regression.
